package.json

```diff
@@ -13,7 +13,8 @@
     "build:test": "cp envConfig/test.env .env && npm run webpack:prod",
     "build:live": "cp envConfig/live.env .env && npm run webpack:prod",
     "cypress:proxy": "export NO_PROXY=localhost,localhost.bbc.com && cypress",
-    "cypress": "npm run cypress:proxy -- run",
+    "cypress": "test -z $CI && npm run cypress:proxy -- run || npm run cypress:ci",
+    "cypress:ci": "npm run cypress:proxy -- run | perl -pe 's/\\x1b\\[[0-9;]*m//g'",
```

Comment:
You've added this but not used it. You mean to use it on travis and on jenkins don't you? I think using it in test:prod:ci script would be the fix.

Reply:
I'm using it in the line above :) (although, looking at that line again, it's wrong... let me fix that)